library/cwm/src/lib/cwm/dialog.rb

```diff
@@ -86,16 +86,18 @@ def cwm_show
       Yast::CWM.show(
         contents,
         caption:        title,
-        back_button:    replace_true(back_button, Yast::Label.BackButton),
-        abort_button:   replace_true(abort_button, Yast::Label.AbortButton),
-        next_button:    replace_true(next_button, Yast::Label.NextButton),
+        back_button:    convert_value(back_button, Yast::Label.BackButton),
+        abort_button:   convert_value(abort_button, Yast::Label.AbortButton),
+        next_button:    convert_value(next_button, Yast::Label.NextButton),
         skip_store_for: skip_store_for
       )
     end
 
-    def replace_true(value, replacement)
-      if value == true
-        replacement
+    def convert_value(value, default)
+      case value
+      when true then default
+      # in CWM empty string means remove button, nil means use default, which is here default value
```

Comment:
I want to explain that CWM::Dialog use nil as "omit" button, but CWM module use "nil" as default value and `""` as omit button.